dev/src/user.clj

```diff
@@ -1,25 +1,33 @@
 (ns user
   (:require
    [environ.core :as env]
-   [humane-are.core :as humane-are]
-   [mb.hawk.assert-exprs]
    [metabase.bootstrap]
-   [metabase.test-runner.assert-exprs]
-   [pjstadig.humane-test-output :as humane-test-output]))
+   [metabase.plugins.classloader :as classloader]
+   [metabase.util :as u]))
 
-;; Initialize Humane Test Output if it's not already initialized. Don't enable humane-test-output when running tests
-;; from the CLI, it breaks diffs. This uses [[env/env]] rather than [[metabase.config]] so we don't load that namespace
-;; before we load [[metabase.bootstrap]]
-(when-not (= (env/env :mb-run-mode) "test")
-  (humane-test-output/activate!))
+;; Wrap these with ignore-exceptions to reduce the "required" deps of this namespace
```

Comment:
without this we get `Caused by: java.io.FileNotFoundException: Could not locate metabase/test_runner/assert_exprs__init.class, metabase/test_runner/assert_exprs.clj or metabase/test_runner/assert_exprs.cljc on classpath. Please check that namespaces with dashes use underscores in the Clojure file name.`

when we call the `:migrate` alias.